### ISSUE TITLE
Allow to customize description

### ISSUE THREAD

**Issue by thedrow** [tomykaira/rspec-parameterized#36]
It be useful to be able to change the description of the test.
`a: 1, b: 2, result: 3` is less useful than

```
plus
  1 + 2
   should equal 3
```

**Comment by joker1007** [comment]
Hmm.. It is not easy.
Because `with_them` arguments cannot get parameters.
`with_them` is evaluated before building parameter list.

I suggest you following codes.

```ruby
describe "sample" do
    with_them do
      context "#{params[:a]} + #{params[:b]}" do
        subject { a + b }

        it { is_expected.to eq answer }
      end
    end
end
```

output:
```
sample
  a: 1, b: 2, answer: 3
    1 + 2
      should eq 3
  a: 5, b: 8, answer: 13
    5 + 8
      should eq 13
  a: 0, b: 0, answer: 0
    0 + 0
      should eq 0
```

This code is a little verbose, but doesn't need more hack.

**Comment by thedrow** [comment]
If you'll allow to provide the message format in the where block that won't be a problem. Just evaluate the block whenever you invoke describe.

**Comment by joker1007** [comment]
I cannot imagine that.
Please tell me sample code snippet.

**Comment by aliaksandr-martsinovich** [comment]
I believe this is fixed in 0.4.0

**Comment by thedrow** [comment]
Yup :)
